docs/constrained.rst

```diff
@@ -15,13 +17,60 @@ set and :math:`\upsilon` are parameter the convex set may depend on.
 Projected gradient
 ------------------
 
+.. autosummary::
+  :toctree: _autosummary
+
+    jaxopt.ProjectedGradient
+
+Instantiating and running the solver
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Projected gradient descent is gradient descent with an additional projection
-onto the constraint set.
+onto the constraint set. Constraints are specified by setting the
+``projection`` argument::
+
+  from jaxopt import ProjectedGradient
+  from jaxopt.projection import projection_non_negative
+
+  pg = ProjectedGradient(fun=fun, projection=projection_non_negative)
+  pg_sol = pg.run(w_init, data=(X, y)).params
+
+Numerous projections are available, see below.
+
+Specifying projection parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some projections have a hyperparameter that can or must be specified.  For
```

Comment:
```suggestion
Projections can optionally have hyperparameters.  For
```

(I find the can or must very confusing, but feel free to use another construction)